tests/queries/0_stateless/01472_toStartOfInterval_disallow_empty_tz_field.sql

```diff
@@ -0,0 +1,20 @@
+SELECT toStartOfDay(toDateTime('2017-12-31 00:00:00', 'UTC'), ''); -- {serverError 43}
+SELECT toStartOfDay(toDateTime('2017-12-31 03:45:00', 'UTC'), 'UTC'); -- success
+
+SELECT toStartOfMonth(toDateTime('2017-12-31 00:00:00', 'UTC'), ''); -- {serverError 43}
```

Comment:
`toStartOfMonth` and `toStartOfQuarter` will still pass with empty time zone and should be fixed, will fix it in next commit.